### ISSUE TITLE
Latex support

### ISSUE THREAD

**Issue by trappedinspacetime** [birros/web-archives#7]
First of all thank you for developing this application. It's useful for offline users.
I downloaded `stackexchange` math zim archive, unfortunately math expressions are not rendered.
Is it possible to use javascript library in this link https://www.mathjax.org/ in `web-archives`?
Regards.

**Comment by birros** [comment]
Thank you for your support 🙂

Unfortunately, I don't really have the time or energy to devote to the development of WebArchives. In addition, I would have to rewrite it from scratch to make it easier to maintain. I will probably do so in several years, when some of the technologies I expect will appear. In the meantime, I'm just fixing the bugs that can be reported and also updating the flatpak manifest each time a new version of GNOME appears.

Concerning your bug, on my side I tested on several **Mathematics Stack Exchange** zim files and I didn't find any problems. I therefore recommend you try to open your zim file using **KiwixDesktop**, this one is the software developed by the same people who develop the `libzim` library used by WebArchives to read the files, and also produce the `zim` files. KiwixDesktop is still in beta and is intended to be multiplatform, so it is developed in QT and is therefore less well integrated with GNOME than WebArchives. Its advantage is that it is maintained by a full-time team, unlike WebArchives which as I told you is an amateur project.

**You can download the KiwixDesktop flatpak in beta version from this address:** https://download.kiwix.org/release/kiwix-desktop/

**If the problem still persists then it is a problem coming from the creation of the zim file, in this case I invite you to open a ticket here:** https://github.com/openzim/sotoki

I close this ticket for the moment, if however the problem did not come from the zim file but from WebArchives in this case I will reopen this ticket.

Have a good day 🖖
